price-feeder/price-feeder.example.toml

```diff
@@ -76,9 +76,9 @@ enable-hostname = true
 enable-hostname-label = true
 enable-service-label = true
 enabled = true
-global-labels = [["chain-id", "umee-local-testnet"]]
+global-labels = [["chain_id", "umee-local-testnet"]]
 service-name = "price-feeder"
-type = "prometheus"
+prometheus-retention-time = 100
```

Comment:
Thought about adding a reference above the `telemetry` block to [the docs](https://pkg.go.dev/github.com/cosmos/cosmos-sdk/telemetry#Config), but we should comment the whole file with docs if we do that

Reply:
But would be a good idea to comment the whole file '-'